src/extension.ts

```diff
@@ -10,14 +10,14 @@ import * as fs from 'fs';
 import * as path from 'path';
 import { MongoExplorer } from './mongo/explorer';
 import { MongoCommands } from './mongo/commands';
-import { Model, Database, Server, IMongoResource, MongoScript, Collection } from './mongo/mongo';
+import { Model, Database, Server, IMongoResource, MongoCommand, Collection } from './mongo/mongo';
```

Comment:
I don't understand your first question. MongoScript and MongoCommand are the same thing. I just renamed it from 'MongoScript' to 'MongoCommand' to be more accurate.

In terms of the UI, we plan to have two options: "Execute Command" and "Execute All Commands". We won't ever call it "Execute Script"